packages/sync-actions/test/utils/create-map-action-group.spec.js

```diff
@@ -0,0 +1,89 @@
+import createMapActionGroup from '../../src/utils/create-map-action-group'
+
+describe('', () => {
```

Comment:
I am not sure why this is empty string.

using `createMapActionGroup` might work since that's the method we are testing.

Reply:
Ups.